commons/src/main/java/org/infinispan/commons/util/AbstractFileLookup.java

```diff
@@ -65,7 +65,7 @@ public InputStream lookupFileStrict(URI uri, ClassLoader cl) throws FileNotFound
          case "jar":
          case "file":
             String uriAsString = uri.toString();
-            String fileName = uriAsString.substring(uriAsString.lastIndexOf(":"));
+            String fileName = uriAsString.substring(uriAsString.lastIndexOf(":") + 1);
```

Comment:
I'd prefer to treat them differently: the 'jar' scheme has its own special syntax, so I'd avoid attempting to collapse parsing logic.
`
case "file":
    return new FileInputStream(new File(uri.getPath()));
case "jar":
   String path = uri.getPath();
   return new FileInputStream(new File(path.substring(path.indexOf(':')+1)));
`